example/index.html

```diff
@@ -29,13 +29,7 @@
     async function startInstance(module, instance) {
         namespace = $('#namespace').val();
 
-        let localKey = ""
         let remoteKey = ""
-
-        if (localStorage.getItem(namespace+":localKey")) {
```

Comment:
It is still used in line 61 which just doesnt show the user the field to enter the passphrase or the server address if the remote key is already known. It just makes the example client a bit nicer, so I decided to leave it. But I guess I can change it to just check if the item is set rather than assigning it to a variable. Will update 👍

Reply:
updated 👍

Reply:
Ah, I didn't see that. But looks nicer the way you did it now.